eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -229,7 +229,7 @@ function Upload-Blobs
     if ($UploadLatest -and ($latestVersion -eq $DocVersion))
     {
         Write-Host "Uploading $($PkgName) to latest folder in $($DocDest)..."
-        & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/latest$($SASKey)" --recursive=true
+        & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/latest$($SASKey)" --recursive=true --cache-control "max-age=300, must-revalidate"
```

Comment:
Don't we also want to add this to our versions file updated? Based on the issue I thought that was the file they were interested in making sure the cache expires.

Reply:
I have already updated the existing ones for temporary fix. I was thinking we might only need to fix newly created one. But after a second though, it still has the chance of overwriting the existing version doc. So I will update the rest places(new and existing ones).